src/components/panel/panel.scss

```diff
@@ -178,3 +166,7 @@
   z-sticky;
   inline-size: fit-content;
 }
+
+[hidden] {
```

Comment:
Maybe this could be something we include in every component somehow?

Reply:
Seems that PR was already merged. I think this was still needed. Might need some investigation. 🔍